sql/src/test/java/io/crate/executor/transport/AlterTableOperationTest.java

```diff
@@ -72,4 +73,19 @@ public void testPrepareRenameTemplateRequests() throws Exception {
 
         assertThat(requests.v2().name(), is(PartitionName.templateName(sourceIdent.schema(), sourceIdent.name())));
     }
+
+    @Test
+    public void testPrepareAlterTableMappingRequest() throws Exception {
+        Map<String, Object> oldMapping = new HashMap<>();
+        oldMapping.put("properties", new HashMap<String, String>() {{ put("foo", "foo"); }});
```

Comment:
not so bad on tests, but in general it is. would be great if you change it to use a MapBuilder instead